src/debug/logger.lisp

```diff
@@ -20,10 +24,12 @@
 ;;     automatic 'conversion' routines which do things like, e.g., discard
 ;;     pointers to objects and retain only their public addresses.
 
-(defun log-entry (&rest initargs &key (logger *logger*) source time entry-type &allow-other-keys)
+(defun log-entry (&rest initargs
+                  &key (logger *logger*) (log-level 0) source time entry-type
+                  &allow-other-keys)
   "Injects a log entry."
   (declare (ignore source entry-type time))
-  (when logger
+  (when (and logger *log-level* (>= log-level *log-level*))
```

Comment:
is this what you mean? “nil will never ever pass this”? why would anyone make a call with nil then? mb `(and logger (or (null *log-level*) (>= log-level *log-level*)))`

Reply:
nil is the default, that's all -- maybe i should just default it to 0